modules/acid/src/test/ts/atomic/TransformationsTest.ts

```diff
@@ -0,0 +1,61 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+
+import * as Transformations from 'ephox/acid/api/colour/Transformations';
+
+describe('browser.acid.api.ConversionsTest', () => {
+
+  it('old tohex convered Red 0', () => {
```

Comment:
`convered` should be `converted` for all of these. Addtionally `old tohex` doesn't make sense without the history here so we shouldn't be saying that. Also don't forget to prefix the message with the Jira number for this.

Here's an example for this one, but you'll need to fix the rest as well.

- [ ] Fix test descriptions

```suggestion
  it('TINY-8163: RGB with 0 red converted to hex', () => {
```